.github/workflows/aas-registry-e2e-test.yml

```diff
@@ -0,0 +1,72 @@
+###############################################################
+# Copyright (c) 2021, 2023 Robert Bosch Manufacturing Solutions GmbH
```

Comment:
i think you don't need to add 2021 in header for all files.

Reply:
Done